TestShell/shell_test.cpp

```diff
@@ -0,0 +1,15 @@
+#include "gmock/gmock.h"
+#include "shell.h"
+
+using namespace testing;
+using namespace std;
+
+class ShellFixture : public Test {
+public:
+
+};
+
+TEST(ShellFixture, commandValidTest) {
```

Comment:
마이너하지만... 
인스턴스 creation Test인데, commandValidTest로 되어 있군요.
이름 변경이 필요하지 않을까 싶은데요.

Reply:
좋은 의견 이십니다. 의도에 맞게 변경하는게 좋겠습니다.
```suggestion
TEST(ShellFixture, createInstanceTest) {
```

Reply:
확인했습니다.